CHANGES.rst

```diff
@@ -36,9 +36,8 @@ Release history
   testing). We also better detect when we can use ``precompute=True``.
   (`#260 <https://github.com/nengo/nengo-loihi/pull/260>`__)
 - ``Simulator`` now includes a ``.timers`` dictionary that records the wall time
-  taken by particular parts of the simulation. Currently, the main timer available is
-  ``"steps"``, which reports the amount of time taken to run the simulation steps
-  (not including connection time).
+  taken by all simulation steps (``"steps"``) and, for models using SNIPs, the time
+  taken inside SNIPs (``"snips"``).
```

Comment:
This isn't quite accurate, and is part of the reason I didn't talk about "snips" here (I made it more for internal use rather than for users). Basically, it measures the total time after the board has been fully initialized/started (i.e. `HardwareInterface.run_steps` is called), in the case that we're using SNIPs. So it's not just "inside SNIPs", it's really all the time on board and chip, just without startup time.

Reply:
I think the key name `"snips"` is probably not the right one then. We'll discuss offline.

Reply:
Decision made in the meeting is to merge these two timers so that we always do our best to only profile the run time (which in most cases is with what we were calling `"steps"` but when using snips is what we were calling `"snips"`). Then, we will also add a timer to measure build time (which is minimal effort) and one to measure startup time. The startup time should capture the difference between `steps` and `snips` that is currently in this PR.

Reply:
Done in 08dbb8f16d2477eb61f49a3e4e9bec01fb872e30